src/Microsoft.AspNet.Identity/IdentityLogger.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.AspNet.Identity
+{
+    public class IdentityLogger
```

Comment:
Is this class an implementation detail or we expect customers or provider writers to use it? If it is an implementation detail we should consider moving it to a different namespace, e.g. Microsoft.AspNet.Identity.Internal (to be consistent with what we are doing with internal implementation details across the framework) or Microsoft.AspNet.Identity.Infraestructure if we think the API is solid and we don't want to discourage customers from using it directly.